app/views/notes/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <table class="table mb-5">
+        <button type="button" class="btn btn-success mb-2 float-right"><%= link_to "Create Note", new_user_note_path, class:'text-light' %></button>
```

Comment:
why creating link inside button?

you can use link_to only by adding button classes to it.

Reply:
done